lib/Room.php

```diff
@@ -203,7 +202,6 @@ public function __construct(Manager $manager,
 								string $password,
 								string $remoteServer,
 								string $remoteToken,
-								int $activeGuests,
```

Comment:
This is a value on the model, please keep it injected and also keep the getter + setter to modify the value on the model (only the DB update part is basically moved to the RoomService